features/fixtures/Gemfile

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
```

Comment:
Did you mean to exclude this file? I'm guessing you had it temporarily to run a command to generate a new Rails 6 app.